include/test/mir/test/gtest_fixes.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2020 Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 or 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Christopher James Halse Rogers <christopher.halse.rogers@canonical.com>
+ */
+
+#ifndef MIR_TEST_GTEST_FIXES_H_
+#define MIR_TEST_GTEST_FIXES_H_
+
+#include "check_gtest_version.h"
+
+#if !GTEST_AT_LEAST(1,9,0)
+#define INSTANTIATE_TEST_SUITE_P(...) INSTANTIATE_TEST_CASE_P(__VA_ARGS__)
+#endif
```

Comment:
I'm not sure this is the best of possible solutions. Alternatives:
1. `#ifndef INSTANTIATE_TEST_SUITE_P`...
2. Add the define in CMake and avoid introducing another header

Reply:
Turns out I can do this in CMake easily enough.